etc/ci_scripts/dump_artifacts/cli.py

```diff
@@ -56,7 +56,7 @@ def main() -> None:
     cwd = os.getcwd()
     timestamp = datetime.now().isoformat(timespec="seconds")
     timestamp = timestamp.replace(":", "-")  # colons in paths are bad
-    artifacts_dir = Path(f"{cwd}/test_artifacts/{compose_project}_{timestamp}")
+    artifacts_dir = Path(f"{cwd}/test_artifacts/{timestamp}")
```

Comment:
I think this isn't enough information, it's still really useful to know whether a set of artifacts is from integration tests or e2e tests. I'd recommend structure like this:

`{yyyy}/{mm}/{dd}/{hh}/{compose_project}_{yyyy}-{mm}-{dd}_{hh}-{mm}-{ss}`

Reply:
actually I'll amend that.. `{yyyy}/{mm}/{dd}/{hh}/{compose_project}`